code/modules/cargo/packs/armory.dm

```diff
@@ -216,23 +205,6 @@
 					/obj/item/clothing/gloves/combat)
 	crate_name = "swat crate"
 
-/datum/supply_pack/security/armory/swattasers //Lesser AEG tbh
-	name = "SWAT tactical tasers Crate"
-	desc = "Contains two tactical energy gun, these guns are able to tase, disable and lethal as well as hold a seclight. Requires Armory access to open."
-	cost = 7000
-	contains = list(/obj/item/gun/energy/e_gun/stun,
-					/obj/item/gun/energy/e_gun/stun)
-	crate_name = "swat taser crate"
-
-/datum/supply_pack/security/armory/woodstock
```

Comment:
Wood stuck only holds iirc 4 in "mag" 1 in chamber, they should be fine to stay, cant even be sawn.